### DISCUSSION TITLE
FEATURE: Choosing protocol in Link Settings

### DISCUSSION THREAD

**Discussion by insssomniac** [GrapesJS/grapesjs#4509] [Ideas]
Currently when we editing a link, we should put to the Settings > Href field whole url with 'https://' part (or without if we want the internal url). But sometimes this approach little complicated for common users, and when they put something like 'google.com' there, they getting 404 because default behavior in that case is to go to the 'https://<site url>/google.com'. 

It would more convenient if the user can choose a protocol from dropdown select options "internal", "https://", "http://", and if the user paste the url with "https://" part, this part will cutted and protocol will choose automatically.

![image](https://user-images.githubusercontent.com/42058438/184325331-c6975c6f-9db9-4bee-a83c-aad6bdccc1fd.png)

**Comment by artf** [comment]
Hi @insssomniac you might be right but for such functionality, I'd prefer to let developers extend the link component and add their traits and logic.
